src/lib/rest/rest.cpp

```diff
@@ -1042,6 +1035,15 @@ static int connectionTreat
   }
 
   // 3. Finally, serve the request (unless an error has occurred)
+  
+  // ServicePath check is delayed to third MHD call 
```

Comment:
Because MHD is not ready to send until there ...
Right?
It all works as long as there are no errors ...
If so, are there any other similar cases?